supervisor/api/audio.py

```diff
@@ -137,7 +137,7 @@ async def set_volume(self, request: web.Request) -> None:
     @api_process
     async def set_mute(self, request: web.Request) -> None:
         """Mute audio volume on stream."""
-        source: StreamType = StreamType(request.match_info.get("source"))
+        source: StreamType = StreamType(request.match_info.get("source", ""))
```

Comment:
As discussed, this really should always be present. For those cases let's use the dictionary syntax:
```suggestion
        source: StreamType = StreamType(request.match_info["source"])
```